pkg/controller/import-controller.go

```diff
@@ -76,6 +84,31 @@ func (ic *ImportController) findImportPodFromCache(pvc *v1.PersistentVolumeClaim
 	return podList[0], nil
 }
 
+func (ic *ImportController) requiresScratchSpace(pvc *v1.PersistentVolumeClaim) bool {
+	scratchRequired := false
+	contentType := getContentType(pvc)
+	// All archive requires scratch space.
+	if contentType == "archive" {
+		scratchRequired = true
+	} else {
+		switch getSource(pvc) {
+		case SourceS3:
+			scratchRequired = true
```

Comment:
I don't think scratch is strictly required for S3

Reply:
No you are right should use the same rules as http, fixed.